understanding/22/target-size-minimum.html

```diff
@@ -104,49 +104,53 @@ <h3>Size requirement</h3>
-              <figcaption>Targets that are 24 by 24 pixels or larger can be arranged closely together. Reducing the target size to 10 by 10 pixels requires spacing/clearance of 14 pixels around each target, meaning that the overall space occupied by the four targets is bigger.</figcaption>
+          <figure id="pointer-target-inline-links">
+            <img src="img/pointer-target-example6-inline-links.png" alt="Inline links in a block of text and in an ordered list (see legend below)" />
+            <figcaption>Two examples show inline text links. The first example has various links in a block of text. The font size is 14px, line height is 16.8px. Depending on the position of links in the text, the vertical offset of links is smaller than 24 CSS pixels. For linked footnote numbers, the horizontal offset is also below 24 CSS pixels. The inline exception allows inline targets to have an offset below the requirement of 24 by 24 CSS pixels.</figcaption>
           </figure>
 
-          <figure id="target-size-minimum-redux">
-            <img src="img/target-size-minimum.png"
-              alt="First diagram: five 24 by 24 pixel targets, arranged in a cross (one target at the center, one target above, one to the right, one below, and one to the left), with no gaps. This passes. Second diagram: reducing the size of the central target to 10 by 10 pixels, while leaving the outer targets unchanged, leads to spacing of only 7 pixels between the central target and the outer targets. This fails. Third diagram: to pass, the outer targets need to be moved further away from the central target 10 by 10 pixel target, with spacing of 14 pixels." />
-            <figcaption>In this (hypothetical) cross arrangement of five 24 by 24 pixel targets, shrinking the central target to 10 by 10 pixels, with spacing of only 7 pixels to the other targets, will fail the Success Criterion. To pass, the outer targets will need to be moved further out, to achieve a spacing of 14 pixels - leading to an increase of the overall space occupied by the five targets.</figcaption>
+          <figure id="pointer-target-stacked-links">
+            <img src="img/pointer-target-example7-stacked-links.png" alt="A bulleted list of stacked links with sufficient and insufficient offset (see legend below)" />
+            <figcaption>The two examples show lists of stacked links as used in navigation, for example in a list of contents of a longer text page. In both cases, the font size is 14px. In the first example, the line height is 25.8px, exceeding the requirement of 24 CSS pixels. In the second example, the stacked links have a line height of just 16.8px. They do not meet the vertical offset requirement of 24 CSS pixels.</figcaption>
```

Comment:
This is not actually true! Because of the wording of inline, the second set of links pass, because it is a numbered list.

Reply:
again this was all part of my botched merge...see previous comments